cms/templates/js/team-member.underscore

```diff
@@ -16,7 +16,7 @@
       <span class="user-username"><%= user.username %></span>
       <span class="user-email">
         <a class="action action-email" href="mailto:<%= user.email %>"
-                title="<%= viewHelpers.format(gettext("send an email message to {email}"), {email: user.email})%>">
+                title="<%= viewHelpers.format(gettext("send an email message to %(email)s"), {email: user.email})%>">
```

Comment:
Note to reviewers: This is caused by the switch to interpolate.

Note also that these "<%=" usages should be changed to "<%-", but I prefer not to mix that into this PR (it will be part of a general bash to fix these issues).